Highrachy/ballers-api
# #172525102 Create user authentication

#### What does this PR do?
This PR authenticates a user and return a valid json data.

#### Description of Task to be completed?
1. Create an authentication middle
2. Get the current User
3. Update status code to use an object

#### How should this be manually tested?
1. Check out into this branch
2. Run `npm install` to update dependencies
3. Start the API server on your local machine `npm run start:dev`
4. Sign in with a valid-user
5. Copy the token and paste in the `Header` section with the title `Authorization`.
6. Visit `http://localhost:3000/api/v1/user/who-am-i`.
7. You should get a valid user JSON object 

#### Screenshot
<kbd>
<img width="1134" alt="Screenshot 2020-05-02 at 2 12 45 PM" src="https://user-images.githubusercontent.com/26963369/80865141-2c49f100-8c7f-11ea-9574-00336fdfefa0.png">
</kbd>
